kubefiles/keycloak/keycloak.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   instances: 1
   extensions:
-    - https://github.com/aerogear/keycloak-metrics-spi/releases/download/1.0.4/keycloak-metrics-spi-1.0.4.jar
+    - https://github.com/aerogear/keycloak-metrics-spi/releases/download/2.5.3/keycloak-metrics-spi-2.5.3.jar
```

Comment:
cool, btw have you tried to remove this thing entirely? maybe we don't need this metrics extension

Reply:
Nope. Just trying to fix for LP ASAP :)